features/element-hiding.json

```diff
@@ -2393,6 +2393,15 @@
                     }
                 ]
             },
+            {
+                "domain": "ncaa.com",
+                "rules": [
+                    {
+                        "selector": ".mobile-persistent-bottom",
+                        "type": "hide"
```

Comment:
@icodebyamanda Just want to double-check – does `hide-empty` not work here?

Reply:
@tagawa I tested it again and only "hide" has worked. Bear in mind tho that I'm testing via chrome dev tools.

Reply:
@icodebyamanda OK, thanks for double-checking.